src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -126,15 +132,93 @@ export const PaginationContainer = styled.div`
 `
 
 export const TabsContainer = styled.div`
```

Comment:
Don't know where exactly the problem is but in implementation the search input goes to the second row on a bigger screen size than in the design

Reply:
not 100% sure this is the case for what you are mentioning here, but I do have noticed our media query breakpoints don't match designs so that causes differences

Reply:
Hmm maybe yeah, we don't have a breakpoint close to `768px` like the design does